calicoctl/commands/node/run.go

```diff
@@ -298,17 +298,30 @@ Description:
 		setNFConntrackMax()
 	}
 
+	// Make sure the calico-node is not already running before we attempt
+	// to start the node.
+	fmt.Println("Removing old calico-node container (if running).")
+	err = exec.Command("docker", "rm", "-f", "calico-node").Run()
+	if err != nil {
+		log.WithError(err).Debug("Unable to remove calico-node container")
+	}
+
 	// Run the docker command.
-	fmt.Println("Running the following command:")
+	fmt.Println("Running the following command to start calico-node:")
 	fmt.Printf("\n%s\n\n", strings.Join(cmd, " "))
+	fmt.Println("If the container image is not available locally, this may take a short amount")
+	fmt.Println("of time to complete.\n")
```

Comment:
hmmm.. not sure how i feel about having 2 separate lines for a single message, what if my terminal size is 60 chars and it wraps, first line would be 2 lines (1 and a half-ish) and then `of time to complete.` in a new line leaving the second half of the line empty. It might look like this:
```
If the container image is not available locally, this may
take a short amount
of time to complete.
```
might be better to just have it in one Println, WDYT?